app/models.py

```diff
@@ -1,9 +1,25 @@
 import requests
 import os
 from flask import json
-from elasticsearch import Elasticsearch
 
-es = Elasticsearch()
+
+api_url = os.getenv('DM_API_URL')
+api_access_token = os.getenv('DM_BUYER_FRONTEND_API_AUTH_TOKEN')
+search_url = os.getenv('DM_SEARCH_API_URL') + "/search"
+search_access_token = os.getenv('DM_BUYER_FRONTEND_SEARCH_API_AUTH_TOKEN')
+
+if api_access_token is None:
+    print('Token must be supplied in DM_BUYER_FRONTEND_API_AUTH_TOKEN')
+    raise Exception("DM_BUYER_FRONTEND_API_AUTH_TOKEN token is not set")
+if api_url is None:
+    print('API URL must be supplied in DM_API_URL')
+    raise Exception("DM_API_URL is not set")
+if search_access_token is None:
+    print('Token must be supplied in DM_BUYER_FRONTEND_SEARCH_API_AUTH_TOKEN')
+    raise Exception("DM_BUYER_FRONTEND_SEARCH_API_AUTH_TOKEN token is not set")
+if search_url is None:
+    print('Search API URL must be supplied in DM_SEARCH_API_URL')
+    raise Exception("DM_SEARCH_API_URL is not set")
```

Comment:
I'm not sure if it's worth doing later rather than right now, but instead of looking up the variables here we could add them to the app config. `app.create_app` already checks the environment for variables matching our config attributes, we could extend it to raise an exception when the value is `None` in the config and the matching environment variable doesn't exist.

We'll then have to pass the config or the required values to the `search_for_services`, but this will make it easier to test.